baseband/mark4/header.py

```diff
@@ -284,13 +290,17 @@ class Mark4Header(Mark4TrackHeader):
               3, 5, 7, 9, 11, 13, 15, 17, 19, 21, 23, 25, 27, 29, 31, 33]])
                  .reshape(1, 16, 2) - 2)}
 
-    def __init__(self, words, ntrack=None, decade=None, verify=True):
+    def __init__(self, words, ntrack=None, decade=None, ref_time=None,
+                 verify=True):
         if words is None:
             self.words = np.zeros((5, ntrack), dtype=np.uint32)
         else:
             self.words = words
         if decade is not None:
             self.decade = decade
+        elif ref_time is not None:
```

Comment:
Looking at this again, I wonder why this is done here; I think it is more logically done in `Mark4TrackHeader`. In that case, here one should do
```
if words is None:
    words = np.zeros((5, ntrack), dtype=np.uint32)
super(Mark4Header, self).__init__(words, decade=decade, ref_time=ref_time, verify=verify)
```